spec/models/game_spec.rb

```diff
@@ -16,4 +16,13 @@
       expect(game.check?('white')).to eq false
     end
   end
+
+  describe 'stalemate method working correctly' do
+    it 'returns true when game is in stalemate' do
+      game = FactoryGirl.create(:game)
+      FactoryGirl.create(:king, horizontal_position: 1, vertical_position: 1, color: 'white', game: game)
```

Comment:
@trustinruss, haha, yea I noticed (and fixed) that as well. The test is modified on my local version and will be pushed once I've finished incorporating the check for whether or not another piece is on the same square that you're checking valid_move for. Good catch though!